src/components/DeploymentScheduleCard.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <p-list-item class="schedule-card">
```

Comment:
The base of the class names here we'd want to match the component name, so `deployment-schedule-card` instead of just `schedule-card`.